docs/apps/interchain-accounts/parameters.md

```diff
@@ -1,53 +1,54 @@
 <!--
-order: 4
+order: 6
 -->
 
 # Parameters
 
 The Interchain Accounts module contains the following on-chain parameters, logically separated for each distinct submodule:
 
-### Controller Submodule Parameters
+## Controller Submodule Parameters
 
 | Key                    | Type | Default Value |
 |------------------------|------|---------------|
 | `ControllerEnabled`    | bool | `true`        |
 
-#### ControllerEnabled
+### ControllerEnabled
 
-The `ControllerEnabled` parameter controls a chains ability to service ICS-27 controller specific logic. This includes the sending of Interchain Accounts packet data as well as the following ICS-26 callback handlers:
+The `ControllerEnabled` parameter controls a chains ability to service ICS27 controller specific logic. This includes the sending of Interchain Accounts packet data as well as the following ICS-26 callback handlers:
```

Comment:
I was trying to find a consistent way to write it to because in same places it's written as `ICS-27` in other places `ICS27` and yet in other places as `ICS 27`. I saw your comment of preferring `ICS 27` or `ICS-27` and I am happy with both, so I will change it back.